tests/performance-test/create_nhsapp_template.py

```diff
@@ -0,0 +1,201 @@
+from locust import FastHttpUser, between, task
+import logging
+import re
+from locust_plugins.transaction_manager import TransactionManager
+from time import time
+from datetime import datetime
+
+class SubmitNHSAppTemplate(FastHttpUser):
+    wait_time = between(5,10)
+
+    def on_start(self):
+        self.tm = TransactionManager()
+
+    def get_credential(self):
+        userName = self.environment.parsed_options.user_name
+        password = self.environment.parsed_options.password
+        return (userName,password)
+
+    def get_headers(self):
+        return {
+            'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.7'
+        }
+
+    def template_headers(self):
+        return {
+            'Accept': 'text/x-component',
+            'Content-type': 'text/plain;charset=UTF-8'
+        }
+
+    @task
+    def landing(self):
+        headers = self.get_headers()
+        credentials = self.get_credential()
+
+        self.tm.start_transaction("A_NHSAPP_01_LandingPage")
+
+        with self.client.get("/templates/create-and-submit-templates", name='A_01_CreateSubmitTemplate', headers=headers, auth=(credentials), catch_response=True) as resp:
+            if resp.status_code == 200:
+                if 'Create and submit a template to NHS Notify' in resp.text:
+                    resp.success()
+                    self.tasks = [self.__class__.start_now]
+                else:
+                    resp.failure("response does not contain header text")
+                    self.tasks = [self.__class__.landing]
+            else:
+                print("Error in login")
+                print(resp.content)
+                self.tasks = [self.__class__.landing]
+
+        self.tm.end_transaction("A_NHSAPP_01_LandingPage")
+
+    def start_now(self):
+        headers = self.get_headers()
+        credentials = self.get_credential()
+
+        self.tm.start_transaction("A_NHSAPP_02_StartNow")
+
+        with self.client.get("/templates/create-template", name='A_02_CreateTemplate', allow_redirects=False, headers=headers, auth=(credentials), catch_response=True) as resp:
```

Comment:
instead of hardcoding the "/templates" base path, it'd be better to define a base path in one place and read the URLs from there because when we move to the new architecture the base path will be different for different branches

eg `self.client.get(f'{base_path}/create-template'` etc

Reply:
Base path is removed. It is part of URL that will be taken at the time of test start.